app/views/candidate_interface/personal_details/nationalities/_shared_form.html.erb

```diff
@@ -0,0 +1,20 @@
+<%= f.govuk_error_summary %>
+
+<h1 class="govuk-heading-xl">
+<%= t('page_titles.nationalities') %>
+</h1>
+
+<%= f.govuk_collection_select :first_nationality, select_nationality_options, :id, :name, label: { text: t('application_form.personal_details.nationality.label'), size: 'm' } %>
```

Comment:
I think you also need to update `nationality-autocomplete.js` to point to the new ids.

Reply:
Yea good spot. Thanks Steve.